storage/reads/array_cursor.gen.go.tmpl

```diff
@@ -350,11 +349,6 @@ func (c *{{.name}}WindowFirstArrayCursor) Next() *cursors.{{.Name}}Array {
 	c.res.Timestamps = c.res.Timestamps[:0]
 	c.res.Values = c.res.Values[:0]
 
-	every := c.every
```

Comment:
why is this removed?

Reply:
This was dead code so I decided to remove it and be explicit about what window first array cursors expect. They should not be passed `every = 0` or `every = math.MaxInt64`. In both cases a regular first array cursor should be (and is currently) used.